src/feedback/data/sagas.js

```diff
@@ -3,7 +3,7 @@ import { put } from 'redux-saga/effects';
 import { addMessage, clearMessages } from './actions';
 import { MESSAGE_TYPES } from './constants';
 
-export function* handleErrors(e, clearExistingMessages) {
+export function* handleErrorFeedback(e, clearExistingMessages) {
```

Comment:
I changed the name of these two functions to indicate they're not just doing all our error handling.  They're doing something much more specific that doesn't involve logging anything!

Reply:
showErrorFeedback?  showErrorsToUser?  Hmm.